src/masonite/stubs/api/AuthenticationController.py

```diff
@@ -0,0 +1,16 @@
+from masonite.controllers import Controller
+from masonite.request import Request
+from masonite.response import Response
+from masonite.authentication import Auth
+
+
+class AuthenticationController(Controller):
+    def auth(self, auth: Auth, request: Request, response: Response):
+        user = auth.attempt(request.input("username"), request.input("password"))
+
+        if user:
+            return {"data": user.generate_jwt()}
+
+        return response.json(
```

Comment:
here you returned JSON and in the middleware we return a view. Couldn't we return json directly or am I missing something ?

Reply:
we could yeah. The view method detects the input type so if you did `.view({dict})` it will just call `.json`. `.view()` actually just wraps the controller method so that how we handle all the different response types you can return

but I agree it should be consistent here

Reply:
yep that's what I was pointing out